arcgis-ios-sdk-samples/Edit data/Add features with contingent values/AddFeaturesContingentValuesViewController.swift

```diff
@@ -0,0 +1,202 @@
+        }
+        
+        super.init(coder: coder)
+        
+        // Load the geodatabase.
+        geodatabase?.load { [weak self] (error) in
+            let result: Result<Void, Error>
+            if let error = error {
+                result = .failure(error)
+            } else {
+                result = .success(())
+            }
+            self?.geodatabaseDidLoad(with: result)
+        }
+    }
+    
+    deinit {
+        if let geodatabase = geodatabase {
+            geodatabase.close()
+            try? FileManager.default.removeItem(at: geodatabase.fileURL)
+        }
+    }
+    
+    /// Called in response to the geodatabase load operation completing.
+    func geodatabaseDidLoad(with result: Result<Void, Error>) {
+        switch result {
+        case .success:
+            // Get and load the first feature table in the geodatabase.
+            featureTable = geodatabase.geodatabaseFeatureTables[0] as AGSArcGISFeatureTable
+            featureTable?.load { [weak self] error in
+                guard let self = self else { return }
+                if let error = error {
+                    self.presentAlert(error: error)
+                } else {
+                    // Create and load the feature layer from the feature table.
+                    let featureLayer = AGSFeatureLayer(featureTable: self.featureTable!)
+                    featureLayer.load { _ in
+                        // Add the feature layer to the map.
+                        self.mapView.map?.operationalLayers.add(featureLayer)
+                        // Set the map's viewpoint to the feature layer's full extent.
+                        let extent = featureLayer.fullExtent
+                        self.mapView.setViewpoint(AGSViewpoint(targetExtent: extent!))
+                        // Add buffer graphics for the feature layer.
+                        self.createBufferGraphics()
+                    }
+                }
+            }
+        case .failure(let error):
+            presentAlert(error: error)
+        }
+    }
+    
+    /// Creates a map with a vector tiled layer basemap.
+    func makeMap() -> AGSMap {
+        // Get the vector tiled layer by name.
+        let fillmoreVectorTiledLayer = AGSArcGISVectorTiledLayer(name: "FillmoreTopographicMap")
+        // Use the vector tiled layer as a basemap.
+        let basemap = AGSBasemap(baseLayer: fillmoreVectorTiledLayer)
+        let map = AGSMap(basemap: basemap)
+        return map
+    }
+    
+    /// Add a single feature to the map.
+    func addFeature(at mapPoint: AGSPoint) {
+        // Create a symbol to represent a bird's nest.
+        let symbol = AGSSimpleMarkerSymbol(style: .circle , color: .black, size: 11)
+        // Add the graphic to the graphics overlay.
+        self.graphicsOverlay.graphics.add(AGSGraphic(geometry: mapPoint, symbol: symbol))
+        // Show the attributes table view.
+        performSegue(withIdentifier: "AddFeature", sender: self)
```

Comment:
Phil might suggest using an `@IBSegueAction` here. See https://github.com/Esri/arcgis-runtime-samples-ios/pull/1038#discussion_r583962813 and https://github.com/Esri/arcgis-runtime-samples-ios/pull/1127#discussion_r773379574 for instance.